DR Strategy Update - 2H 2023 (#4332)
MicrosoftDocs/architecture-center: docs/data-guide/disaster-recovery/dr-for-azure-data-platform-deploy-this-scenario-content.md

- [Azure Service Health](/azure/service-health/service-health-overview) within their Azure management portal will provide the latest updates
    - If there are issues accessing Service Health, refer to the [Azure Status page](https://status.azure.com/en-us/status)
    - If there are ever issues accessing the Status page, go to @AzureSupport on X (formerly Twitter)
- If impact/issues don’t match the incident (or persist after mitigation), then [contact support](https://www.aka.ms/AzurePortalSupportRequest) to raise a service support ticket

For Power BI

- The [Service Health](https://portal.office.com/AdminPortal/Home) page within their Microsoft 365 admin center will provide the latest updates
    - If there are issues accessing Service Health, refer to the [Microsoft 365 status page](https://status.office.com)

>     - If impact/issues don’t match the incident (or if issues persist after mitigation), they should contact support to raise a [service support ticket](https://learn.microsoft.com/power-bi/support/create-support-ticket)

Reviewer: ```suggestion
    - If impact/issues don't match the incident (or if issues persist after mitigation), you should raise a [service support ticket](/power-bi/support/create-support-ticket).
```
